includes/Core/Authentication/Clients/OAuth_Client.php

```diff
@@ -731,10 +734,14 @@ function( $scope ) {
 		 *
 		 * @since 1.3.0
 		 * @since 1.6.0 The $token_response parameter was added.
+		 * @since n.e.x.t The $scopes parameter was added.
+		 * @since n.e.x.t The $prevous_scopes parameter was added.s
```

Comment:
There's a few typos on the second one here, but also these can be added in the same annotation. We should only have 1 since tag per version.
```suggestion
		 * @since n.e.x.t The $scopes and $previous_scopes parameters were added.
```

Reply:
👍. Thanks for fixing my typo.